pkg/udevevent/monitor.go

```diff
@@ -122,3 +164,61 @@ func Monitor() {
 		}
 	}
 }
+
+func Subscribe(eventTypes ...UdevEventType) *Subscription {
```

Comment:
Add comments

Reply:
Why do we need to return the Subscription here?

Reply:
we need the subscription in the `Unsubscribe` call